layers/meta-balena-genericx86/recipes-core/images/balena-image.inc

```diff
@@ -30,4 +30,7 @@ write_mbr() {
 
 IMAGE_POSTPROCESS_COMMAND:append = " write_mbr; "
 
-IMAGE_INSTALL:append = " efibootmgr"
+IMAGE_INSTALL:append = " \
+	efibootmgr \
+	haveged \
```

Comment:
@jakogut I think this commit needs more explanation. Why is the hardware RNG not working?

Reply:
To push this PR getting merged let's merge it without haveged installed and we'll see after. As Alex said, we already have rngd which is responsible for entropy generation but the problem might be different iirc.